BreastCancer/TabBarControllers/Dashboard/APHDashboardViewController.m

```diff
@@ -163,58 +163,46 @@ - (void)prepareScoringObjects {
+                                                 latestOnly:NO];
     self.exerciseScoring.customMinimumPoint = 1.0;
     self.exerciseScoring.customMaximumPoint = 5.0;
     
-    self.sleepScoring = [[APCScoring alloc] initWithTask:@"3-APHMoodSurvey-7259AC18-D711-47A6-ADBD-6CFCECDED1DF"
-                                                  numberOfDays:-kNumberOfDaysToDisplay
-                                                      valueKey:@"moodsurvey105"
-                                                     dataKey:nil
-                                                     sortKey:nil
-                                                  groupBy:APHTimelineGroupDay];
+    self.sleepScoring = [[APCScoring alloc] initWithTask:kDailySurveyIdentifier
+                                            numberOfDays:-kNumberOfDaysToDisplay
+                                                valueKey:@"moodsurvey105"
+                                              latestOnly:NO];
     self.sleepScoring.customMinimumPoint = 1.0;
     self.sleepScoring.customMaximumPoint = 5.0;
     
-    self.cognitiveScoring = [[APCScoring alloc] initWithTask:@"3-APHMoodSurvey-7259AC18-D711-47A6-ADBD-6CFCECDED1DF"
-                                                  numberOfDays:-kNumberOfDaysToDisplay
-                                                      valueKey:@"moodsurvey102"
-                                                         dataKey:nil
-                                                         sortKey:nil
-                                                      groupBy:APHTimelineGroupDay];
+    self.cognitiveScoring = [[APCScoring alloc] initWithTask:kDailySurveyIdentifier
+                                                numberOfDays:-kNumberOfDaysToDisplay
+                                                    valueKey:@"moodsurvey102"
+                                                  latestOnly:NO];
     self.cognitiveScoring.customMinimumPoint = 1.0;
     self.cognitiveScoring.customMaximumPoint = 5.0;
     
-    self.customScoring = [[APCScoring alloc] initWithTask:@"3-APHMoodSurvey-7259AC18-D711-47A6-ADBD-6CFCECDED1DF"
-                                                    numberOfDays:-kNumberOfDaysToDisplay
-                                                        valueKey:@"moodsurvey107"
-                                                         dataKey:nil
-                                                         sortKey:nil
-                                                         groupBy:APHTimelineGroupDay];
+    self.customScoring = [[APCScoring alloc] initWithTask:kDailySurveyIdentifier
+                                             numberOfDays:-kNumberOfDaysToDisplay
+                                                 valueKey:@"moodsurvey107"
+                                               latestOnly:NO];
     self.customScoring.customMinimumPoint = 1.0;
     self.customScoring.customMaximumPoint = 5.0;
```

Comment:
Create a utility method takes `valueKey` (e.g. `@"moodsurvey107"` ) as its input, since all other parameters are same across these various types of scoring.